phase-4-explainer.md

```diff
@@ -0,0 +1,175 @@
+### Phase 4: Full Introspection of an Accessibility Tree
+
+The **Computed Accessibility Tree** API will allow authors to access
+the full computed accessibility tree -
+all computed properties for the accessibility node associated with each DOM element,
+plus the ability to walk the computed tree structure including virtual nodes.
```

Comment:
virtual nodes -> [virtual nodes](https://github.com/dot-miniscule/aom/blob/phase4-outline/explainer.md#phase-3-virtual-accessibility-nodes), and [potentially](#open-questions) other objects not expressed as DOM nodes, such as `::before` content.

Reply:
Done.